### ISSUE TITLE
Joining Analytics Sessions page : demo sends back 404

### ISSUE THREAD

**Issue by Antoinebr** [ampproject/docs#1854]
## Description of the issue found on ampproject.org

Please provide:

- Page link: 
https://amp.dev/static/samples/linker/demo-page.html

https://github.com/ampproject/docs/blob/ae01c3340fa6762ab92c6f2aa3b11eeacacd7f8e/examples/source/advertising-analytics/Joining_Analytics_Sessions.html#L15

- Description of the documentation issue

## How do we reproduce the issue?

Please provide the steps to reproduce the issue:

When you click on "Please click to continue" on https://amp.dev/static/samples/linker/demo-page.html you hit a 404

The current link is :
```
https://amp.dev/linker/destination-page.html
``` 

It should be : 

```
https://amp.dev/static/samples/linker/destination-page.html
```

**Comment by Antoinebr** [comment]
This PR should fix it #1855

**Comment by matthiasrohmer** [comment]
Now fixed, thanks again!
